Browser_Actions/Browser_ActionTests.cs

```diff
@@ -0,0 +1,126 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SeleniumExtras.WaitHelpers;
+using OpenQA.Selenium.Interactions;
+using System.Xml.Linq;
+using System.Drawing;
+using OpenQA.Selenium.Chrome;
+
+namespace Browser_Actions
+{
+    [TestFixture]
+    public class Browser_ActionTests : BaseTest_BA
+    {
+        [Test]
+        public void OpenNewWindowHandleTabs()
+        {
+            Setup();
+            string originalWindow = Driver.CurrentWindowHandle;
+            Assert.AreEqual(Driver.WindowHandles.Count, 1);
+            var elementMultipleWindows = Driver.FindElement(By.CssSelector("[href*='windows']"));
+            elementMultipleWindows.Click();
+            var elementClickHere = Driver.FindElement(By.XPath("//*[@class='example']//a[1]"));
```

Comment:
Try to use locators without [1] [2] if it possible. Usually test is stable, try to use it

Reply:
in some cases removed, in e.g. Headless mode test  input [1], left to select specific element